src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+
+    private int radius;
+
+    public Circle(int radius, String figureColor) {
+        super(figureColor);
+        this.radius = radius;
+    }
+
+    public double getArea() {
+        double area = Math.PI * radius * radius;
+        return area;
```

Comment:
```suggestion
        return Math.PI * radius * radius;
```
the same for other figures